### DISCUSSION TITLE
Stockfish 500 points in chess.com 06/12/2023

### DISCUSSION THREAD

**Discussion by Ilosttothechampion** [official-stockfish/Stockfish#4907] [Q&A]
In the chess.com whoever has 500 points beats the stockfish 8 16 in lichess.org and has 10000 points in chess.com than people play much better who's best on the planet 
Why i tied with stockfish 6 2300 points and this beat me easily 

I drew against stockfish 6 on 29/11/2023 
https://lichess.org/6kYUjo1fJTE4 
This one kept the two rooks and the queen and almost the 7 pawns and capture my pieces
chess.com/live/game/94199797136

**Comment by MammaUauua** [comment]
Stockfish level 6 is different from Stockfish 6.
